spec/rails_helper.rb

```diff
@@ -2,16 +2,17 @@
 ENV["RAILS_ENV"] ||= 'test'
 require 'spec_helper'
 require File.expand_path("../../config/environment", __FILE__)
-require 'rspec/rails'
 
 # coverage with simplecov and codeclimate
 require 'simplecov'
 require "codeclimate-test-reporter"
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+SimpleCov.formatters = [
   SimpleCov::Formatter::HTMLFormatter,
   CodeClimate::TestReporter::Formatter
 ]
-SimpleCov.start "rails"
+SimpleCov.start 'rails'
+
+require 'rspec/rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.